### PR TITLE
scope-hoisting-closure excercise solution

### PR DESCRIPTION
**Pull request by CuongNgMan**

### CODE REVIEW

File: scope-hoisting-closure/excercise-1.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  }
+  return cal;
+}
+
+const rAvg = runningAverage();
+
+console.log(rAvg(10));
+console.log(rAvg(11));
+console.log(rAvg(12));
+console.log(rAvg(13));
```
Comment (viktordavidovich) [inline scope-hoisting-closure/excercise-1.js:19]:
```
function runningAverage() {
  const accumSpeed = [];

  function cal(speed) {
    accumSpeed.push(speed);

    const len = accumSpeed.length;
    let sum = accumSpeed.reduce(function (previousValue, currentValue) {
      return (previousValue + currentValue)
    }, 0)

    return sum / len;
  }
  return cal;
}

const rAvg = runningAverage();

console.log(rAvg(1));
console.log(rAvg(3));
console.log(rAvg(7));
console.log(rAvg(10));
console.log(rAvg(11));
console.log(rAvg(12));
console.log(rAvg(13));
```

Comment (viktordavidovich) [reply scope-hoisting-closure/excercise-1.js:19]: I got your solution, but I believe the idea of task was to implement common solution to calculate average speed provided by user. In addition, again, as far as I understood the task, we need to accumulate the given speed to calculate average result. Please, try the code above.

File: scope-hoisting-closure/excersice-2.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * 2.	Write	a	sum	function	which	will	 work	properly	when	invoked	using	syntax	below.
+ */
+
+function add(a, b) {
+  return a + b;
+}
```
Comment (viktordavidovich) [inline scope-hoisting-closure/excersice-2.js:7]: I would move this function inside function sum to keep global object clean. 
I believe this is best practice.

Comment (viktordavidovich) [reply scope-hoisting-closure/excersice-2.js:7]: Overall, great solution!

**Review by viktordavidovich — APPROVED** [review]
Please review comments, overall all looks great!

File: scope-hoisting-closure/excercise-1.js (hunk 1 above)
Comment (CuongNgMan) [reply scope-hoisting-closure/excercise-1.js:19]: Great feedback. I misunderstood the results of the example.

**Pull request merged**